cmake/Protobuf.cmake

```diff
@@ -1,4 +1,6 @@
 macro(p4c_obtain_protobuf)
+  set(P4C_PROTOBUF_VERSION "25.3")
+  set(P4C_PROTOC_VERSION "4.25")
```

Comment:
is `P4C_PROTOC_VERSION` used at all?

Reply:
No, I was hoping to use it for `https://github.com/p4lang/p4c/pull/4543/files#diff-08181e83a64a6a99c0661f941d7b457ae42e4313384e857565f4d9545fd82903R22` but that code still seems to break. Let me try it one more time.

Reply:
I just tried, and it looks like adding the version simply breaks the find_package command. I removed the version specifier for now.